### PR TITLE
Adds a new drip option - cloth handwraps!

### PR DESCRIPTION
**Pull request by Sadhorizon**
## What Does This PR Do
Adds a new drip option - cloth handwraps.
Available in loadout for one point, can be crafted out of two cloth (like footwraps).

## Why It's Good For The Game
I always wondered - why is it not a thing already? Idea is pretty drippy if you ask me!

## Images of changes
obraz
obraz

## Testing
Spawned in, spun around as every species, checking if sprites look fine.

## Changelog
:cl:
add: Added cloth handwraps. Available in loadout. Can also be crafted with cloth.
/:cl:

### CODE REVIEW

**Comment by Mitchs98** [comment]
Real Rocky vibes up in here.

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -273,6 +273,7 @@ GLOBAL_LIST_INIT(cloth_recipes, list (
 		new /datum/stack_recipe("white scarf", /obj/item/clothing/accessory/scarf/white, 1),
 		new /datum/stack_recipe("white shoes", /obj/item/clothing/shoes/white, 2),
 		new /datum/stack_recipe("cloth footwraps", /obj/item/clothing/shoes/footwraps, 2),
+		new /datum/stack_recipe("cloth handwraps", /obj/item/clothing/gloves/handwraps, 2),
 		null,
 		new /datum/stack_recipe("fingerless gloves", /obj/item/clothing/gloves/fingerless, 1),
 		new /datum/stack_recipe("white gloves", /obj/item/clothing/gloves/color/white, 3),
```
Comment (Henri215) [inline code/game/objects/items/stacks/sheets/sheet_types.dm:279]:
```suggestion
		null,
		new /datum/stack_recipe("cloth handwraps", /obj/item/clothing/gloves/handwraps, 2),
		new /datum/stack_recipe("fingerless gloves", /obj/item/clothing/gloves/fingerless, 1),
		new /datum/stack_recipe("white gloves", /obj/item/clothing/gloves/color/white, 3),
```
Better keep it close to the other gloves.

**Review by DGamerL — APPROVED** [review]
Code is fine

**Review by S34NW — APPROVED** [review]

**Comment by Sadhorizon** [comment]
GAMING

**Pull request merged**